.travis.yml

```diff
@@ -8,9 +8,7 @@ before_install:
   # Fix for Ubuntu Xenial apt-daily.service triggering
   # https://unix.stackexchange.com/questions/315502/how-to-disable-apt-daily-service-on-ubuntu-cloud-vm-image
   - |
-    sudo systemctl stop apt-daily.service &&
-    sudo systemctl kill --kill-who=all apt-daily.service &&
-    while ! (systemctl list-units --all apt-daily.service | fgrep -q dead) ; do
+    while sudo fuser /var/{lib/{dpkg,apt/lists},cache/apt/archives}/lock >/dev/null 2>&1; do
```

Comment:
Why redirect stderr?

Reply:
To avoid `Specified filename <filename> does not exist`. We are only interested in the exit code

Reply:
Ah, of course, I misread the code and forgot that it was redirecting both to `/dev/null`